client/my-sites/importer/style.scss

```diff
@@ -1,335 +1,3 @@
-	text-align: center;
-
-	input {
-		visibility: hidden;
-	}
-
-	.accessible-focus &:focus {
-		border-color: var( --color-neutral-200 );
-	}
-}
-
-.importer__upload-content {
-	position: absolute;
-	top: 50%;
-	left: 0;
-	right: 0;
-	text-align: center;
-	transform: translateY( -50% );
-}
-
-.importer__upload-progress,
-.importer__import-progress {
-	&.progress-bar {
-		width: 80%;
-	}
-
-	.progress-bar__progress {
-		background-color: var( --color-accent );
-	}
-
-	&.is-complete .progress-bar__progress {
-		background-color: var( --color-success );
-	}
-}
-
-.importer__import-spinner {
-	position: absolute;
-	left: 50%;
-	transform: translateX( -50% );
-}
-
-.importer__start {
-	float: none !important;
-}
-
-.importer__importing-pane {
-	text-align: center;
-}
-
-.importer__status-message {
-	font-style: italic;
-	color: var( --color-neutral-light );
-}
-
-.importer__mapping-header {
-	border-bottom: 1px solid var( --color-neutral-light );
-	margin-bottom: 16px;
-	padding-bottom: 8px;
-
-	font-size: 11px;
-	color: var( --color-neutral-light );
-
-	@include clear-fix;
-}
-
-.importer__mapping-description {
-	margin-bottom: 16px;
-
-	strong {
-		white-space: nowrap;
-	}
-}
-
-.importer__mapping-source-title {
-	float: left;
-	display: inline-block;
-}
-
-.importer__mapping-target-title {
-	float: right;
-	display: inline-block;
-}
-
-.importer__author-mapping {
-	height: 58px;
-
-	.author-selector__author-toggle {
-		width: 45%;
-		padding-top: 4px;
-		margin-right: -7px;
-		float: right;
-	}
-
-	div.user {
-		display: inline;
-	}
-}
-
-.importer__source-author {
-	width: 45%;
-	height: 18px;
-	padding-top: 7px;
-	float: left;
-	display: inline-block;
-	color: var( --color-neutral-700 );
-
-	text-align: left;
-	font-size: 14px;
-	font-weight: 600;
-}
-
-.importer__mapping-relation {
-	float: left;
-	color: var( --color-neutral-200 );
-}
-
-.site-importer__site-importer-pane {
-	position: relative;
-	margin-bottom: 24px;
-
-	& > div {
-		margin-bottom: 10px;
-	}
-}
-
-.site-importer__site-importer-url-input {
-	display: flex;
-	margin-bottom: 1.5em;
-
-	.form-text-input {
-		flex-grow: 1;
-		flex-shrink: 1;
-	}
-
-	button {
-		flex-grow: 0;
-		flex-shrink: 0;
-	}
-}
-
-.site-importer__site-importer-endpoint-select {
-	width: 100%;
-}
-
-.site-importer__site-importer-example-domain {
-	font-family: monospace;
-	color: var( --color-accent );
-}
-
-.site-importer__site-importer-confirm-site-pane-container {
-	display: flex;
-	align-content: center;
-	align-items: flex-start;
-	flex-wrap: wrap;
-	width: 100%;
-	margin-bottom: 18px;
-
-	button {
-		float: none;
-		flex-grow: 0;
-		flex-shrink: 0;
-
-		&:first-of-type {
-			margin-left: 0;
-		}
-	}
-}
-
-.site-importer__site-importer-confirm-site-label {
-	flex-grow: 1;
-	display: flex;
-	flex-direction: column;
-	margin-right: 10px;
-}
-
-.site-importer__source-url {
-	margin-top: 4px;
-	margin-bottom: 8px;
-}
-
 .site-importer__site-importer-confirm-site-pane-container-text {
```

Comment:
@jsnajdr I'm not really sure what to do with this one, so I left it here for now. It's not in use. Should I leave it here or remove it? Or move it, but where?

Reply:
It's indeed not used, so let's remove it.